docs/actions.md

```diff
@@ -233,7 +234,10 @@ Each action invocation results in an activation record which contains the follow
 - `response`: A dictionary that defines the following keys
   - `status`: The activation result, which might be one of the following values:
     - *"success"*: the action invocation completed successfully.
-    - *"application error"*: the action invocation was successful, but the action returned an error value on purpose, for instance because a precondition on the arguments was not met.
+    - *"application error"*: the action was invoked, but returned an error value on purpose, for instance because a precondition on the arguments was not met. This status code is also returned under specific conditions such as:
+      - wrong docker container name
+      - initialization of a function failed
+      - action timed out
```

Comment:
some of these are about to change; see #3941.

Reply:
OK. Will change it.